nats-server/configuration/securing_nats/authorization.md

```diff
@@ -26,6 +28,22 @@ The `permission` map provides additional properties for configuring a `permissio
 
 **Important Note** NATS Authorizations can be _allow lists_, _deny lists_, or both. It is important to not break request/reply patterns. In some cases \(as shown below\) you need to add rules as above with Alice and Bob for the `_INBOX.>` pattern. If an unauthorized client publishes or attempts to subscribe to a subject that has not been _allow listed_, the action fails and is logged at the server, and an error message is returned to the client.
 
+## Allow Responses Map
+
+The `allow_responses` option dynamically allows publishing to reply subjects, this works well for service responders.
+When set to `true`, excepting any `publish` permissions, implicitly all publish permissions are denied unless it is the reply subject in a request received by the client.
+
+The `allow_responses` map also allow you to configure a maximum number of responses and how long to allow the dynamic permission after a message is received.
+
+| Property | Description |
+| :--- | :--- |
+|  `max` | The maximum number of response messages that can be returned to a request. |
+| `expires` | The duration of time to allow a response. Values such as `1s`, `1m`, `1h` (1 second, minute, hour) etc can be specified. |
```

Comment:
expires I think better described as, the amount of time the permission is valid.